wicker/core/column_files.py

```diff
@@ -90,8 +90,8 @@ class ColumnBytesFileWriter:
 
     def __init__(
         self,
-        storage: S3DataStorage = S3DataStorage(),
-        s3_path_factory: S3PathFactory = S3PathFactory(),
+        storage: AbstractDataStorage = S3DataStorage(),
+        s3_path_factory: WickerPathFactory = S3PathFactory(),  # S3-specific naming kept for backwards compatibility.
```

Comment:
Above ^^^^ it is too bad that in the `ColumnBytesFileWriter` class `__init__` function that the argument `s3_path_factory` was given an S3-specific name. I can't think of a good way to make a backwards-compatible change (to a name like `path_factory`, so I just left it as `s3_path_factory`). Let me know if you have another (backwards-compatible) suggestion!